auxx/Main.hs

```diff
@@ -102,7 +102,7 @@ action opts@AuxxOptions {..} command = do
         printAction "Mode: with-config"
         CLI.printInfoOnStart aoCommonNodeArgs ntpConfig
         (nodeParams, tempDbUsed) <-
-            correctNodeParams opts =<< CLI.getNodeParams loggerName cArgs nArgs
+            (correctNodeParams opts . fst) =<< CLI.getNodeParams loggerName cArgs nArgs
```

Comment:
`getNodeParams` now returns `Maybe SscParams` which is almost always used alongside the `NodeParams`. Here's an exception.